### DISCUSSION TITLE
No matching distribution found for xformers==0.0.27.post2

### DISCUSSION THREAD

**Discussion by Yogesh-DevHub** [bmaltais/kohya_ss#2786] [Q&A]
ERROR: Could not find a version that satisfies the requirement xformers==0.0.27.post2 (from versions: none)
ERROR: No matching distribution found for xformers==0.0.27.post2

tried installing VC_redist.X64 - 14.29.30153 not worked, any other solution?

**Comment by PGCRT** [comment]
Same error

**Comment by MegaCocos** [comment]
Same error here. Upgrading to xformers 0.027.post2 manually it installs Torch 2.4.0 as well and then says that it is not compatible with other scripts. Reinstalling Kohya_SS (Branch Flux) it changes again Torch 2.4.0 for Torch 2.4.1 that is not compatible with xformers 0.027.post2 and then gives errors. If anybody wants to investigate i can give more details. Windows 10

**Reply by YadeFr** [reply]
> Same error here. Upgrading to xformers 0.027.post2 manually it installs Torch 2.4.0 as well and then says that it is not compatible with other scripts. Reinstalling Kohya_SS (Branch Flux) it changes again Torch 2.4.0 for Torch 2.4.1 that is not compatible with xformers 0.027.post2 and then gives errors. If anybody wants to investigate i can give more details. Windows 10

Same observation for me, has anyone managed to install the application from Flux branch ?

**Reply by Yogesh-DevHub** [reply]
try flux gym in pinokio

**Reply by PGCRT** [reply]
Or use comfyUI Trainer it's more flexible and it doesn't try to resize captions .txt files like FluxGym (leading to an error), plus you will stay in the comfy environment if you are used to it ,) 
https://civitai.com/models/743615/easy-lora-trainer
